PSKoans/Koans/Constructs and Patterns/AboutRegularExpressions.Koans.ps1

```diff
@@ -0,0 +1,468 @@
+                character. Normally if you saw a lowercase "n" in a regex, it would just mean
+                "literally the letter n". When it's preceded by a backslash, however, that "n" takes
+                on a special meaning. In this case, \n matches new lines.
+    
+                There are many more regex symbols that are single letters preceded by a backslash.
+                In fact, the backslash is probably the single most important character in all of
+                regex.
+            #>
+    
+            $multiLine = @"
+They might look
+similar and look
+soft, but you should
+never confuse a
+wild cougar for
+a domestic cat
+"@
+            $catPattern = '____\n'
+            $multiLine -match $catPattern | Should -BeTrue
+        }
+    }
+
+    Context '\d and \D' {
+        It 'matches digits and non-digits' {
+            <#
+                The \d symbol matches digits (the numbers 0 - 9 inclusive), while the symbol \D
+                matches anything OTHER THAN a digit. Yes, regular expressions are case sensitive.
+                This "lower case for affirmative, upper case for negative" pattern is common among
+                regex symbols as you'll discover as you carry on.
+            #>
+    
+            '__' -match '\d' | Should -BeTrue
+            '__' -match '\d\d\d' | Should -BeTrue
+            '__' -match '\D' | Should -BeTrue
+        }
+    }
+
+    Context '\w and \W' {
+        It 'matches word characters' {
+            <#
+                \w matches word characters which are letters, numbers, and underscores. The \W
+                symbol matches anything else.
+            #>
+            '***' -match '____' | Should -BeTrue
+            'warmth' -match '____' | Should -BeTrue
+        }
+    }
+
+    Context '^ (caret) and $ (dollar sign)' {
+        It 'matches the start and end of lines' {
+            <#
+                The ^ symbol matches the start of a line, which is not normally represented in
+                English with a character. The $ represents the end of a line.
+            #>
+    
+            $startAndEnd = '____'
+            $startAndEnd -match '^a' | Should -BeTrue
+            $startAndEnd -match 'z$' | Should -BeTrue
+        }
+    }
+
+    Context '\s and \S' {
+        It 'matches (non-) whitespace characters' {
+            <#
+                \s matches whitespace characters (space, tab, etc.) while \S matches anything other
+                than whitespace
+            #>
+    
+            # Enter either \s or \S - which do you think will work?
+            'Room to grow' -match '____' | Should -BeTrue
+
+            # Careful to only fill in a value for the '____', not the '    '
+            '    ' -match '____' | Should -BeFalse
+
+            'accordian' -match '____' | Should -BeFalse
+            'accordian' -match '____' | Should -BeTrue
+        }
+    }
```

Comment:
Same pattern here; newline before the `It`(s) and split these out into separate `It` statements.